tests/test_sn_flux_est.py

```diff
@@ -71,22 +82,28 @@ def test_find_t_sys_gain():
     pulsar_1= "J2234+2114"
     obsid_2=1222697776
     pulsar_2= "J2330-2005"
-    md_1 = mwa_metadb_utils.get_common_obs_metadata(obsid_1) #this is to speed up the tests
-    md_2 = mwa_metadb_utils.get_common_obs_metadata(obsid_2)
+
+    md_1 = md_dict[str(obsid_1)][0]
+    full_md_1 = md_dict[str(obsid_1)][1]
+    md_2 = md_dict[str(obsid_2)][0]
+    full_md_2 = md_dict[str(obsid_2)][1]
+
     q_1 = psrqpy.QueryATNF(psrs=[pulsar_1], loadfromdb=ATNF_LOC)
     q_2 = psrqpy.QueryATNF(psrs=[pulsar_2], loadfromdb=ATNF_LOC)
     test_cases = []
-    test_cases.append((pulsar_1, obsid_1, 1223042887, 600, q_1, md_1,\
-                325.17339020027805, 6.5034678040055613, 0.12547449408456718, 0.095633857616224477))
-    test_cases.append((pulsar_2, obsid_2, None, None, q_2, md_2,\
-                295.73944858721245, 5.9147889717442492, 0.29127750035795497, 0.049367868667752078))
+    #test_cases.append((pulsar_1, obsid_1, 1223042887, q_1, md_1, full_md_1,\
+    #            325.16928459135534, 6.5033856918271065, 0.1405071004856564, 0.1065230048864121))
+    
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing whitespace](https://app.codacy.com/manual/CIRA-Pulsars-and-Transients-Group/vcstools/pullRequest?prid=5199130)